goon-universe/app.js

```diff
@@ -225,7 +225,7 @@ router.route('/scenarios/initialAction')
             res.end()
         }
         else{
-        db.getInitActions(scenarioID, function(result){
+        db.getInitActionPageQuestionsAndChoices(scenarioID, function(result){
```

Comment:
Querying `GET /scenarios/initialAction` with `scenarioID: 1, studentID: 1` results in an unhandled exception being thrown.

Reply:
DT's implementation of the getInitActionPageQuestionsAndChoices takes questionID as an argument along with scenarioID. I believe this is the issue. Also this function does not require studentID. @rasdy @rlee287

Reply:
Function now does callback on empty object (if question and options do not exist) or on JSON object with `question` and `mcq_choices` fields. I will ping both of you on Slack once I finish these changes and push them.

Reply:
This should be fixed now.